src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/Anchira.kt

```diff
@@ -193,26 +202,42 @@ class Anchira : HttpSource(), ConfigurableSource {
 
     // Details
 
-    override fun mangaDetailsRequest(manga: SManga) =
-        GET("$libraryUrl/${getPathFromUrl(manga.url)}", headers)
+    override fun mangaDetailsRequest(manga: SManga): Request {
+        return if (manga.url.startsWith("?")) {
+            GET(libraryUrl + manga.url, headers)
+        } else {
+            GET("$libraryUrl/${getPathFromUrl(manga.url)}", headers)
+        }
+    }
 
     override fun mangaDetailsParse(response: Response): SManga {
-        val data = json.decodeFromString<Entry>(response.body.string())
+        return if (response.request.url.pathSegments.count() == apiUrl.toHttpUrl().pathSegments.count()) {
+            val manga = latestUpdatesParse(response).mangas.first()
+            val query = response.request.url.queryParameter("s")
+            manga.apply {
+                url = "?${response.request.url.query}"
+                description = "Bundled from $query"
+                title = "[Bundle] $query"
```

Comment:
title should maybe contain series title instead of the query

Reply:
That can be arranged. I went for this instead because another way I use it is for artist collections